src/components/pages/Chat/AddRoomModal/PrivacySelect.tsx

```diff
@@ -0,0 +1,185 @@
+    },
+]
+
+const customStyles = {
+    control: (provided: any) => ({
+        ...provided,
+        border: 'none',
+        display: 'flex',
+        flexDirection: 'row',
+        backgroundColor: '#DEE6EE',
+        height: '56px',
+        borderRadius: '8px',
+        alignItems: 'center',
+        paddingLeft: '20px',
+        paddingRight: '10px',
+        marginBottom: '15px',
+    }),
+    indicatorSeparator: () => ({ display: 'none' }),
+    menu: (provided: any) => ({
+        ...provided,
+        borderRadius: '10px',
+        boxShadow: '0px 0px 10px rgba(0, 0, 0, 0.1)',
+        marginTop: '8px',
+        zIndex: '100000',
+    }),
+    option: (provided: any) => ({
+        ...provided,
+        backgroundColor: 'white',
+    }),
+}
+
+const OptionContainer = styled.div`
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+`
+
+const LabelContainer = styled.div`
+    display: flex;
+    flex-direction: column;
+    margin-right: auto;
+`
+
+const IconContainer = styled.div`
+    border-radius: 50%;
+    background-color: #f1f4f7;
+    width: 30px;
+    height: 30px;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    margin-right: 10px;
+`
+
+const CheckIcon = styled.img`
+    width: 12px;
+    height: 9px;
+    margin-left: auto;
+`
+
+const ControlIcon = styled.div`
+    color: #59799C:
+`
+
+const Option = (props: any) => {
+    return (
+        <components.Option {...props}>
+            <OptionContainer>
+                <IconContainer>
+                    <img src={props.data.icon} />
+                </IconContainer>
+                <div
+                    style={{
+                        display: 'flex',
+                        flexDirection: 'row',
+                        alignItems: 'center',
+                        justifyContent: 'space-between',
+                        width: '100%',
+                    }}
+                >
+                    <LabelContainer>
+                        <p
+                            style={{
+                                color: '#36404E',
+                                fontSize: '14px',
+                                margin: '0px',
+                            }}
+                        >
+                            {props.data.label}
+                        </p>
+                        <p
+                            style={{
+                                color: '#59799C',
+                                fontSize: '12px',
+                                margin: '0px',
+                            }}
+                        >
+                            {props.data.subLabel}
+                        </p>
+                    </LabelContainer>
+                    {props.isSelected && <CheckIcon src={checkIcon} />}
+                </div>
+            </OptionContainer>
+        </components.Option>
+    )
+}
+
+const Control = ({ children, ...props }: any) => {
+    console.log(props.selectProps.value)
```

Comment:
@its-edmund let's avoid logging stuff in production. cc @bearni95 

An option would be: a custom `log(…args)` that checks `NODE_ENV === 'development'` and proceeds to `console.log(…args)`